demo/napari_gui/napari_magicgui.py

```diff
@@ -0,0 +1,36 @@
+# Inspired by
+# https://github.com/pr4deepr/pyclesperanto_prototype/blob/master/napari_clij_widget.py
+import napari
+import pyclesperanto_prototype as cle
+from magicgui import magicgui
+from napari.layers import Image, Labels
+
+@magicgui(auto_call=True)
+def process_image(input: Image, sigma: float = 5) -> Labels:
+    if input:
+        # push image to GPU
+        input = cle.push_zyx(input.data)
```

Comment:
Does `cle` autopush? And if so, can we take advantage of that here? ;) And if not, can we raise an issue? =P

Reply:
it auto-pushes, but it doesn't do push_zyx. I'll create and issue on this: https://github.com/clEsperanto/pyclesperanto_prototype/issues/49